### ISSUE TITLE
Prohibited from submitting transcriptions (Davy Notebooks Project)

### ISSUE THREAD

**Issue by snblickhan** [zooniverse/Panoptes-Front-End#5463]
Note: this report is from a volunteer on the [Davy Notebooks Project](https://www.zooniverse.org/projects/humphrydavy/davy-notebooks-project) (#9006).

[Report](https://www.zooniverse.org/projects/humphrydavy/davy-notebooks-project/talk/2586/1056485?comment=1739377) from Talk:

Some volunteers having trouble submitting their classifications. "Next" button is deactivated and cursor turns into "prohibited" symbol: 🚫

This may be an expected behavior, as we've noticed it's occurring when the workflow is more than 90% complete. However, probably worth thinking long-term about whether there's a possible adaptation for this for projects with time-consuming classification tasks (like transcription), as we don't want wasted volunteer effort.

TL;DR: possibly a bug, possibly normal retirement behavior that has disproportionately negative effects for transcription projects.

25/7/2019 update:
A volunteer has reported that this bug seems to be due to 'phantom underlines':

> I get the next box greyed out when there are phantom red underlines. It says there are more underlines than are actually visible. The underline tool is very sticky and sometimes creates lines that aren't visible. Unless you locate them and fill in some text the next box remains greyed out.

I recreated this issue by underlining a section of text, then simply clicking out of the sub-task text entry popup when it appeared. You can't click "OK" in the sub-text popup if you haven't entered any text into the field, but you CAN just click out of the box and the line will remain, without any associated text data being entered. If any of these 'empty' lines are on the page, the "Next" box remains greyed out.

**Comment by snblickhan** [comment]
@mcbouslog and @srallen probably good to have on your radar as we work on text tool effort (this instance is mark line + text subtask).

**Comment by srallen** [comment]
The subtask is required, but it is unclear which line is missing the required annotation
